### PR TITLE
Edits to training during to resolve build errors

### PR DESCRIPTION
**Pull request by cbw36**

### CODE REVIEW

File: exercises/Perception-Driven_Manipulation/template_ws/src/robot_config/urdf/ur5.urdf.xacro [hunk 1]
```diff
@@ -1,70 +1,9 @@
 <?xml version="1.0"?>
-<robot xmlns:xacro="http://ros.org/wiki/xacro">
```
Comment (cbw36) [inline exercises/Perception-Driven_Manipulation/template_ws/src/robot_config/urdf/ur5.urdf.xacro]: Was receiving the error: Undefined parameters [hw_interface] when instantiating macro: ur_arm_transmission.

To resolve the error, substituted the existing ur5.urdf.xacro with `universal_robot/ur_description/urdf/ur5.urdf/xacro`

File: gh_pages/_downloads/ros-industrial-training-setup.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ sudo apt update -y
 # ROS1 install
 sudo apt install -y ros-melodic-desktop
 sudo apt install -y python-wstool python-catkin-tools
-sudo apt install -y ros-melodic-perception ros-melodic-urdf-tutorials ros-melodic-moveit
```
Comment (cbw36) [inline gh_pages/_downloads/ros-industrial-training-setup.sh]: The package `ros-melodic-urdf-tutorials` does not exist. The correct package name is `ros-melodic-urdf-tutorial`, but it is already installed as a dependancy on another package.

File: gh_pages/_source/session4/Descartes-Path-Planning.md [hunk 3]
```diff
@@ -161,10 +161,12 @@ With the Descartes node completed, we now want to invoke its logic by adding a n
  1. Create a new `setup.launch` file (in `workcell_support` package) that brings up everything except your workcell_node:
 
     ``` xml
-    <include file="$(find myworkcell_moveit_config)/launch/myworkcell_planning_execution.launch"/>
-    <node name="fake_ar_publisher" pkg="fake_ar_publisher" type="fake_ar_publisher_node" />
-    <node name="vision_node" type="vision_node" pkg="myworkcell_core" output="screen"/>
-    <node name="descartes_node" type="descartes_node" pkg="myworkcell_core" output="screen"/>
+    <launch>
```
Comment (cbw36) [inline gh_pages/_source/session4/Descartes-Path-Planning.md:164]: Not sure if it is assumed the individual will know to include <launch></launch>

File: gh_pages/_source/demo1/Inspect-the-package.md [hunk 4]
```diff
@@ -10,6 +10,7 @@ catkin init
 ```
 
 ## Download debian dependencies
+>Clone the ROS-I repo [universal_robot](https://github.com/ros-industrial/universal_robot) into the src directory of `perception_driven_ws`. Switch to the `melodic-devel` branch.
```
Comment (marip8) [inline gh_pages/_source/demo1/Inspect-the-package.md]: `kinetic-devel` is actually the default "master" branch of the UR repository. I would amend this to say:

>Clone the [universal_robot repository](https://github.com/ros-industrial/universal_robot) into the `src` directory of `perception_driven_ws`

**Review by marip8 — CHANGES_REQUESTED** [review]

**Comment by marip8** [comment]
@cbw36 we removed the custom UR5 xacro from this exercise in #315. Can you rebase this branch to remove those now-obsolete changes?

**Comment by cbw36** [comment]
Just pushed after rebasing my branch. Let me know if I need to change anything else

**Comment by marip8** [comment]
Please completely remove the custom `ur5.urdf.xacro` file from this demo. It seems like we missed it in #315. We should edit the places where it is used by referencing the UR5 URDF from the official repository instead.

**Review by marip8 — APPROVED** [review]

**Pull request closed without merging**